.github/dependabot.yml

```diff
@@ -1,16 +1,20 @@
 version: 2
 updates:
-  - package-ecosystem: "gomod"
-    directory: "/"
+  - package-ecosystem: gomod
+    directory: /
     schedule:
-      interval: "weekly"
+      interval: monthly
```

Comment:
A monthly dependency update cadence feels like a sensible balance to me, but obviously open for discussion/opinions